test/lib/ansible_test/_data/requirements/constraints.txt

```diff
@@ -1,3 +1,4 @@
+resolvelib >= 0.5.3, < 0.6.0  # keep in sync with `requirements.txt`
```

Comment:
@mattclay I wasn't sure if I was supposed to put it in any specific part of this file. Is this location fine?